src/SignalR/common/Protocols.MessagePack/src/Protocol/MessagePackHubProtocolWorker.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#pragma warning disable IDE0005 // This file is shared across multiple projects making it ugly to ignore unused usings
```

Comment:
Do you mean we don't have consistent implicit `using`s in projects using this file❔